src/desktop/apps/auction_lots/index.coffee

```diff
@@ -10,6 +10,5 @@ app = module.exports = express()
 app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
-app.get '/artist/:id/auction-results', timeout('25s'), routes.artist
```

Comment:
Due to this being mounted after, this wasn't getting hit, but this is a sign we should clean up code here. We used to have a separate app implementation for auction results, vs the tabbed view we have now.

Now that this app is mounted ahead, we need to remove this.